src/styles/variables.scss

```diff
@@ -0,0 +1,22 @@
+@mixin position($left) {
+  position: absolute;
+  left: $left;
```

Comment:
Useless property, your arrows are already in the middle of clock since you used display: flex and made center aligning for entire clock div.